rocAL/rocAL/source/loaders/image/image_read_and_decode.cpp

```diff
@@ -99,6 +99,12 @@ ImageReadAndDecode::create(ReaderConfig reader_config, DecoderConfig decoder_con
         }
     }
     _reader = create_reader(reader_config);
+
```

Comment:
There is no need to calculate this here. This has be decided by the mastergraph and send it to decoder via DecoderConfig

Reply:
Resolved in the latest commit